### PR TITLE
Supports creating an ACM certificate with DNS validation

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

File: tests/create_certificate/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+locals {
+  test_id = length(data.terraform_remote_state.prereq.outputs) > 0 ? data.terraform_remote_state.prereq.outputs.random_string.result : ""
+
+  zone_name   = "cloudarmor.io"
```
Comment (allez-allez-allez) [inline tests/create_certificate/main.tf]: should this be a variable? other users might be testing using different `zone_name`

Comment (lorengordon) [reply tests/create_certificate/main.tf]: I don't think we actually have a way to pass variables into the tests at the moment, with `make test`... I guess using `TF_VAR_<var>` might work?

Comment (lorengordon) [reply tests/create_certificate/main.tf]: Went ahead and added the var, just with a default so figuring out how to actually pass vars through the test framework can be a future problem....

**Comment by lorengordon** [comment]
Results of `make test`...

```
PASS
ok      terraform-aws-tardigrade-acm/tests      240.056s
```

File: tests/create_certificate/main.tf (hunk 1 above)
Comment (allez-allez-allez) [reply tests/create_certificate/main.tf]: 👍

**Pull request merged**